test/acceptance_test.rb

```diff
@@ -9,58 +8,96 @@ def test_expected_results
 
     expected = [
```

Comment:
Do we think it's rounding errors causing this minor discrepancy?

Reply:
This particular change is related to the separation of `stocks` and `bonds` on the primary account holder's holdings; since each account now accrues interest separately, the fractional cents that may have added up to one cent on the combined value now gets discarded. I switched to `.round` instead of `.floor` when calculating interest to mitigate the difference, but the discrepancy here is what was left.

However, this test is currently failing on this branch due to an additional discrepancy introduced by the inflating int refactor. It's still pretty close, but I'd like to understand exactly what the numbers *should* be and where the discrepancy comes from before we merge.